feat: add new rules for passwords

riderx opened the pull request:
I just started to use Formkit yesterday. It was a wonderful experience but found out the validation for password is a bit light.
So I made this PR who is WIP.
What I added in rules:
- uppercase 
- lowercase
- modified all apha rule to use unicode regex : https://www.regular-expressions.info/unicode.html
- create contain rules, this rule check if at least one match happen.

I tried to run the test in local and that didn't work.
I have yarn in the main folder and then in the sub folder.
But if I do `yesy test` I got:
    SyntaxError: Cannot use import statement outside a module
 On every test.
 
 Could you please help me ?
 Then i will try to work on the translations.

justin-schroeder commented:
Thanks for the PR, You'll need to run a build before you can run tests, just `yarn build all`.

One other thing that needs to happen — if you submit new rules we'll need validation messages in the i18n package as well, ideally for each of the various languages. You can use google translate or chatgpt or something like that to produce them as long as they seem plausible :)

justin-schroeder commented on packages/rules/src/alpha.ts:
```diff
@@ -8,7 +8,7 @@ import { FormKitValidationRule } from '@formkit/validation'
 const alpha: FormKitValidationRule = function ({ value }, set = 'default') {
   const sets = {
-    default: /^[a-zA-ZÀ-ÖØ-öø-ÿĄąĆćČčĎďĘęĚěŁłŃńŇňŘřŚśŠšŤťŮůŹźŻŽžż]+$/,
+    default: /^[\p{Lu}\p{L}]+$/,
```
Don’t forget to add the unicode flag on these 👍 (switching to Unicode is a good suggestion by the way)
```suggestion
    default: /^[\p{Lu}\p{L}]+$/u,
```

riderx commented:
Thanks for the fast answer!
I fixed your point where `u` was missing!
I also added test cases for all the new rules.
Furthermore, I was able to build all and the rules, but still not able to run the test.
I use node 19.

I check the translation now

riderx commented:
@justin-schroeder I did the translation in English and French.
It could be cool to have a tool do automatize it with deepl API for example, seems super long to do the rest

sashamilenkovic commented:
@riderx Thanks. We'll look into this; we can take care of the translations.

riderx commented:
Thanks a lot ! @sashamilenkovic if you can help me running the test i would like to be sure at least the test pass ^^

justin-schroeder commented:
We’ve merged this into #640 where we are iterating on it. Closing this one in favor of that one 👍
